### DISCUSSION TITLE
Bi-directional field error

### DISCUSSION THREAD

**Discussion by KoenRijpstra** [pods-framework/pods#6832] [Q&A]
When i save a custom post type with a Bi-directional field i get the following error:
`The Franchise field is required and cannot be removed by the Vestigingen field`

The field shows 1 connected post but in the `_pods_vestigingen` meta field there are 2 post ids. One of those ids doesn't exist anymore in the wp_posts table. Is it possible that if I remove an item the relation field won't get updated?

Running version 2.3.18

**Comment by Shelob9** [comment]
Sounds like you need to, at least temporarily, disable the required option on the field at the other end.

Have you seen [the page on how relationship fields are saved](http://pods.io/docs/database-reference/database-storage-relationship-fields/)? The way it works is not very obvious so you might want to make sure you're reading that table right.

**Comment by KoenRijpstra** [comment]
Disabling the required option resolves the problem. But the field needs to be required.

I tried to replicate the issue by creating a new relation and removing the related item in the backend but that works as expected. I have no idea how this ever happened in the first place :confused: 

For now i will disable the require option and save all the items and enable the require option again. Thanks for your help.

**Comment by Shelob9** [comment]
@KoenRijpstra I'm glad to help and to hear you got this resolved.

**Comment by lougreenwood** [comment]
I'm finding myself getting this issue, my steps to reproduce are:

* Have 2 Pods linked bi-directionally
* Updating Pod A triggers the creation of some record in Pod B
* Pod B requires a value for it's bi-d link back to Pod A
* The required filed is set when automatically creating the record in Pod B, but the `...field is required and cannot be removed by the...` error is shown
* Disabling required field on Pod B resolves the issue.

**Comment by JoryHogeveen** [comment]
Got the same problem here with a project, though it doesn't happen all the time... trying to find out when it does and when it doesn't.
